scripts/stage-docs.sh

```diff
@@ -159,7 +162,7 @@ gh repo set-default "$(git remote get-url "${REPOSITORY}")"
 
 echo "Creating pull request to publish release documentation to website."
 # Open PR to merge branch to `main` in `kroxylicious/kroxylicious.github.io`
-BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website"
+BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website. Remember to replace the container image SHAs in `_data/release/[this-version].yaml` once they are available, before merging!"
```

Comment:
```suggestion
BODY="Prepare ${RELEASE_TAG} release documentation for publishing to website. Remember to replace the container image SHAs in `_data/release/${RELEASE_VERSION}.yaml` once they are available, before merging!"
```

Shell expansion should work here so we can reference the specific file (obvisouly replace `${RELEASE_VERSION}` with `${RELEASE_TAG}` if approrpaite